src/main/java/com/sap/oss/phosphor/fosstars/data/github/GithubInnerSourceTopicDataProvider.java

```diff
@@ -0,0 +1,54 @@
+package com.sap.oss.phosphor.fosstars.data.github;
+
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.HAS_INNERSOURCE_TOPIC;
+
+import com.sap.oss.phosphor.fosstars.model.Feature;
+import com.sap.oss.phosphor.fosstars.model.Value;
+import com.sap.oss.phosphor.fosstars.model.ValueSet;
+import com.sap.oss.phosphor.fosstars.model.other.Utils;
```

Comment:
```suggestion
import static com.sap.oss.phosphor.fosstars.model.other.Utils.setOf;
```
Just to align with the other classes